### PR TITLE
Adding quickstart for localpod

### PR DESCRIPTION
**Pull request by eadgbear**
## 🗣 Description

Adding new quickstart for the new `localpod` data connector

## 🤔 Concerns

- We might need to better explain the architecture behind use case for this, but that's probably better done in the docs.

### CODE REVIEW

File: localpod/spicepod.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        enabled: true
+        refresh_check_interval: 15s
+        refresh_mode: full
+    - from: localpod:time_series
+      name: local_time_series
+      acceleration:
+        enabled: true
+        engine: duckdb
+        mode: file
+        refresh_check_interval: 10s
```
Comment (ewgenius) [inline localpod/spicepod.yaml]:
```suggestion
```

Comment (ewgenius) [reply localpod/spicepod.yaml]: Child spicepod won't refresh on interval when it is synchronized with parent

File: localpod/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+        enabled: true
+        refresh_check_interval: 15s
+        refresh_mode: full
+    - from: localpod:time_series
+      name: local_time_series
+      acceleration:
+        enabled: true
+        engine: duckdb
+        mode: file
+        refresh_check_interval: 10s
```
Comment (ewgenius) [inline localpod/README.md]:
```suggestion
```

**Comment by Sevenannn** [comment]
We can add a link to localpod documentation to show extra information (e.g. requirement for `refresh_mode: full`) for localpod connector, otherwise look good to me

**Review by Sevenannn — APPROVED** [review]

**Pull request merged**